.eslintrc

```diff
@@ -0,0 +1,34 @@
+{
+  "root": true,
+  "extends": "airbnb-base",
+  "env": {
+    "node": true,
+    "es6": true,
+    "mocha": true,
+    "jasmine": true
+  },
+  "rules": {
+    "one-var": 0,
+    "one-var-declaration-per-line": 0,
+    "new-cap": 0,
+    "consistent-return": 0,
+    "no-param-reassign": 0,
+    "import/no-unresolved": 0,
+    "comma-dangle": 0,
+    "curly": ["error", "multi-line"],
+    "no-shadow": ["error", { "allow": ["req", "done", "res", "err"] }],
+    "valid-jsdoc": ["error", {
```

Comment:
I told you to change this line to this `"valid-jsdoc": [1, {`, which you initially did, why did you revert to `"valid-jsdoc": ["error", {` ?

Reply:
Sorry, the change was on another branch. This branch was created and checked out of develop branch before the jsdoc branch was created. But I'll modify and fix it now